cli/sync_service/download.go

```diff
@@ -55,6 +55,8 @@ func ObjectDownLoad(org string, userPw string, objType string, objId string, fil
 					filePath = filePath + "/"
 				}
 				fileName = fmt.Sprintf("%s%s_%s", filePath, objType, objId)
+			} else {
+				fileName = filePath
```

Comment:
I think you will need to update the code starting at https://github.com/open-horizon/anax/blob/master/cli/sync_service/download.go#L62 in order to handle the case where the file already exists, including a new message and a flag to overwrite the existing file as per the issue description.

Reply:
Sure. Updated PR with adding the new flag.
Previously I was only fixing a bug (that's first part of issue - "At the very least the error msg should say the file already exists"), so after my previous change user would receive that message. I thought that if this issue has an `bug` label we'll need only fix the bug, and if we need to implement a new flat - there will be one more `feature` issue for that.

Reply:
I understand. In this project, we don't make a strong distinction between bug and feature, especially when it comes to user facing interfaces like the CLI.